library/system/test/fs_snapshot_test.rb

```diff
@@ -118,6 +121,46 @@ def logger
           expect(snapshot).to be(dummy_snapshot)
         end
       end
+
+      context "when a cleanup strategy is set" do
+        let(:output) { { "stdout" => "2", "exit" => 0 } }
+        let(:dummy_snapshot) { double("snapshot") }
+        let(:snapshot_command) { CREATE_SINGLE_SNAPSHOT + OPTION_CLEANUP_NUMBER }
+
+        it "returns the created snapshot" do
```

Comment:
when you run `rspec --format doc` it sounds bit strange, that all when have same result. More important here is to mention that strategy is used like 
```
it "creates snapshot with given strategy and returns the created snapshot" do
 ...
end
```
and similar below, so it is obvious how cleanup strategy affect behavior.